scripts/GenPDFlist.py

```diff
@@ -27,27 +27,27 @@ def dictCompare(s):
   header = headerText.readlines()
 
 allFiles = []
-for p in Path(musicFolder).rglob('*.pdf'):
+for p in Path(musicFolder).rglob('*.[Pp][Dd][Ff]'):
```

Comment:
The rglob pattern allows what is often called a character class - the square brackets will match any character in the list, so this pattern means to match a file extension that is either 'P' or 'p' followed by 'D' or 'd', etc.